src/repository/opamGit.ml

```diff
@@ -231,6 +231,18 @@ module VCS : OpamVCS.VCS = struct
       OpamProcess.cleanup ~force:true r; Done true
     | r -> OpamSystem.process_error r
 
+  let modified_files repo_root =
+    git repo_root ~verbose:false [ "status" ; "--short" ] @@> fun r ->
+    OpamSystem.raise_on_process_error r;
+    let files =
+      OpamStd.List.filter_map (fun line ->
+          match OpamStd.String.split line ' ' with
+          | ("A" | "M")::file::[]
+          | ("R"|"RM"|"C"|"CM")::_::"->"::file::[] -> Some file
```

Comment:
Can a file be `copied` and `modified`?

Reply:
I believe so, yes... After a certain threshold of modifications, it will be detected as removed and created. IIRC, this is computed by git after the fact for display, but it doesn't really make a difference to it.